src/components/SelectionList/BaseListItem.js

```diff
@@ -83,6 +83,14 @@ function BaseListItem({
                     )}
                     <ListItem
                         item={item}
+                        textStyle={[
+                            styles.optionDisplayName,
+                            isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText,
+                            item.isSelected && styles.sidebarLinkTextBold,
+                            isUserItem || item.isSelected ? styles.sidebarLinkTextBold : null,
```

Comment:
```suggestion
                            (isUserItem || item.isSelected) && styles.sidebarLinkTextBold,
```
NAB

Reply:
Done

you suggested this change to be consistent with the existing code right? Or is there another reason for this change? just curious

Reply:
Good catch ... reverted